# *********************
# * Velocity mode   0 *
# * Position mode   1 *
# *********************/
uint8 mode
